src/s1/s1/static/stylesheets/scss/components/_s1_header.scss

```diff
@@ -13,38 +10,6 @@
     }
 }
 
-.s1-header {
-    margin-top: rem(50px);
-    margin-bottom: rem(10px);
-
-    position: relative;
-}
```

Comment:
Why was this removed? It's used in `src/s1/s1/static/js/Packages/S1/Context/header.html`.